op-guide/configuration.md

```diff
@@ -14,249 +14,189 @@ The default TiDB ports are 4000 for client requests and 10080 for status report.
 ### `--binlog-socket`
 
 - The TiDB services use the unix socket file for internal connections, such as the PUMP service
-- Default: ``
-- You can use "/tmp/pump.sock" to accept the communication of PUMP unix socket file.
+- Default: ""
+- You can use “/tmp/pump.sock” to accept the communication of PUMP unix socket file.
 
-### `--cross-join`
+### `--config`
 
-- To enable (true) or disable (false) the cross join without any equal conditions
-- Default: true
-- The value can be `true` or `false`. By default, `true` is to enable `join` without any equal conditions (the `Where` field). If you set the value to `false`, the server refuses to run the `join` statement.
+- The configuration file
+- Default: ""
+- If you have specified the configuration file, TiDB reads the configuration file. If the corresponding configuration also exists in the command line flags, TiDB uses the configuration in the command line flags to cover that in the configuration file. For detailed configuration information, see [tidb-config-file.md](tidb-config-file.md)
+    ```
 
 ### `--host`
 
 - The host address that the TiDB server monitors
-- Default: "0.0.0.0"
+- Default: “0.0.0.0”
 - The TiDB server monitors this address.
-- The "0.0.0.0" monitors all network cards. If you have multiple network cards, specify the network card that provides service, such as 192.168.100.113.
-
-### `--join-concurrency int`
-
-- The number of goroutine when `join-concurrency` executes `join` concurrently
-- Default: 5
-- The number depends on the amount of data and data distribution, usually the larger the better, and a larger number means a larger CPU overhead.
+- The “0.0.0.0” monitors all network cards by default. If you have multiple network cards, specify the network card that provides service, such as 192.168.100.113.
 
 ### `-L`
 
-+ The log level
-+ Default: "info"
-+ You can choose from debug, info, warn, error, or fatal.
+- The log level
+- Default: "info"
+- You can choose from "debug", "info", "warn", "error", or "fatal".
 
-### `--lease`
+### `--log-file`
 
-+ The schema lease time in seconds
-+ Default: "10"
-+ This is the schema lease time that is used in online schema changes. The value will affect the DDL statement running time. Do not change it unless you understand the internal mechanism.
+- The log file
+- Default: ""
+- If this flag is not set, logs are written to "stderr". Otherwise, logs are stored in the log file which will be automatically rotated every day.
```

Comment:
- written -> output
- "Otherwise" -> "If this flag is set"
- "logs are stored in the log file which will be automatically rotated every day" -> "logs are output to the corresponding file, which is automatically rotated in the early morning every day, and the previous file is renamed as a backup."